sdk/security/keyvault/azadmin/backup/utils_test.go

```diff
@@ -56,27 +52,32 @@ func run(m *testing.M) int {
 	}
 
 	if recording.GetRecordMode() == recording.PlaybackMode {
-		credential = &FakeCredential{}
+		credential = &azcred.Fake{}
```

Comment:
You can call `azcred.New` unconditionally, it [considers the mode](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/internal/test/credential/credential.go#L23-L24)

Reply:
fixed in [b2c793e](https://github.com/Azure/azure-sdk-for-go/pull/23026/commits/b2c793eea24af689f2230d8717a136230c5bf862)